src/columnresizing.js

```diff
@@ -5,7 +5,7 @@ import {TableMap} from './tablemap';
 import {TableView, updateColumns} from './tableview';
 import {tableNodeTypes} from './schema';
 import {findParentNodeOfTypeClosestToPos} from 'prosemirror-utils';
-import {types} from './columnsTypes/types.config';
+import {columnTypesMap} from './columnsTypes/types.config';
```

Comment:
@OfirSimhi1612 please note that refactor, 5 different places in the code that do this `types.find((type) => type.id === colType)` against one place that actually used the types as an array